src/popup/router/pages/More.vue

```diff
@@ -67,12 +121,92 @@ export default {
   margin: 0 auto;
 
   .navigation {
-    padding: 12px;
-    display: flex;
-    flex-wrap: wrap;
+    width: 100%;
 
     .box-button {
-      margin: 12px;
+      margin: 8px;
+      padding: 8px 12px;
+      width: auto;
+      height: 48px;
+      border-radius: 10px;
+      justify-content: space-between;
+      flex-direction: row;
+      color: rgba(variables.$color-white, 0.95);
+      font-weight: 300;
+      -webkit-transition: 0.4s;
+      transition: 0.4s;
+      opacity: 0.75;
+      letter-spacing: 0.5px;
+
+      &:hover {
+        color: rgba(variables.$color-white, 1);
+      }
+
+      .box {
+        display: inline-flex;
+        align-items: center;
+
+        svg {
+          color: variables.$color-white;
+        }
+
+        .setting-icon {
+          margin-right: 8px;
```

Comment:
I think it would be better to avoid setting right padding based on the icon size to align the labels. Maybe the button labels could have a big padding on the left to accommodate the space for icons and the icons would be position there with position absolute?

Or wrap all icons with a wrapper that would be little bigger and center the icons inside?